tests/plugins/filebrowser/manual/uploadckfinder.html

```diff
@@ -9,6 +9,10 @@ <h2>Inline editor</h2>
 </div>
 
 <script>
+	if ( !CKEDITOR.fileTools.isFileUploadSupported ) {
```

Comment:
This checking causes the test to not run on all non-IE browsers due to the error:

> Uncaught TypeError: Cannot read property 'isFileUploadSupported' of undefined